### ISSUE TITLE
ModuleNotFoundError: No module named 'scipy.special._cdflib'

### ISSUE THREAD

**Issue by snowuyl** [pyinstaller/pyinstaller#8454]
## Description of the issue
pyinstaller can generate main.exe successfully. But main.exe failed to run with the following error message.
ModuleNotFoundError: No module named 'scipy.special._cdflib'

### Context information (for bug reports)
* Output of `pyinstaller --version`: ```6.4.0```
* Version of Python: 3.11.9
* Platform:  Windows 11
* How you installed Python:  Windows store

### A minimal example program which shows the error
Please unzip [pyHrvTest.zip](https://github.com/pyinstaller/pyinstaller/files/15228974/pyHrvTest.zip) and run deploy.bat from command prompt to generate main.exe. And run main.exe.

### Stacktrace / full error message
```
Traceback (most recent call last):
  File "main.py", line 2, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "pyhrvtest\hr.py", line 1, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "pyhrv\__init__.py", line 36, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "pyhrv\hrv.py", line 41, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "biosppy\__init__.py", line 19, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "biosppy\signals\__init__.py", line 23, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "biosppy\signals\acc.py", line 27, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "biosppy\plotting.py", line 28, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "biosppy\signals\tools.py", line 22, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "scipy\signal\__init__.py", line 307, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "scipy\signal\windows\__init__.py", line 42, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "scipy\signal\windows\_windows.py", line 7, in <module>
  File "scipy\__init__.py", line 147, in __getattr__
  File "importlib\__init__.py", line 126, in import_module
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "scipy\special\__init__.py", line 777, in <module>
  File "scipy\\special\\_ufuncs.pyx", line 1, in init scipy.special._ufuncs
ModuleNotFoundError: No module named 'scipy.special._cdflib'
```

**Comment by snowuyl** [comment]
[pyinstaller_generate_pyhrv_log.txt](https://github.com/pyinstaller/pyinstaller/files/15228945/pyinstaller_generate_pyhrv_log.txt)

**Comment by snowuyl** [comment]
[pyHrvTest.zip](https://github.com/pyinstaller/pyinstaller/files/15228974/pyHrvTest.zip)

**Comment by rokm** [comment]
This was fixed in v6.6.0.
